DM-1892: LwM2M - New 'writeb' operation (#758)
SoftwareAG/c8y-docs: content/protocol-integration/lwm2m-bundle/shell-commands.md

<td align="left">Writes value to a resource on the device.</td>
</tr>
<tr>
<td align="left">cwrite /&lt;objectID&gt;/&lt;instanceID&gt;/&lt;resourceID&gt; &lt;value&gt; [/&lt;objectID&gt;/&lt;instanceID&gt;/&lt;resourceID&gt; &lt;value&gt;]</td>
<td align="center">1.1</td>
<td align="left">Composite writes of one or more values to a resource on the device. The data will be written to the listed resource paths in a single request</td>
</tr>
<tr>
<td align="left">writeb /&lt;objectID&gt;/&lt;instanceID&gt;/&lt;resourceID&gt; &lt;hexadecimal-string&gt; <br>
<b>OR</b> <br>
writeb /&lt;objectID&gt;/&lt;instanceID&gt;/&lt;resourceID&gt; binary:&lt;binary-file-id&gt;</td>
<td align="center">1.0,1.1</td>
<td align="left">Writes binary data represented as a hex string to a resource on the device. The representation must be an even number of characters.

> e.g. writeb /3442/0/150 010A0B020F <br><br>

Reviewer: ```suggestion
For example: writeb /3442/0/150 010A0B020F.<br><br>
```

> Writes binary data to a resource on the device from a file uploaded to the Cumulocity platform. The 'binary-file-id' is the object id that has already been uploaded to the Cumulocity platform.

Reviewer: ```suggestion
Writes binary data to a resource on the device from a file uploaded to the {{< product-c8y-iot >}} platform. The 'binary-file-id' is the object ID that has already been uploaded to the {{< product-c8y-iot >}} platform.
```

> It must be added with the prefix 'binary:' so that the agent reads the file's data and writes it to the device.

Reviewer: ```suggestion
Adding the prefix 'binary:' lets the agent read the file's data and write it to the device.
```

> e.g. writeb /3442/0/150 binary:12345

Reviewer: ```suggestion
For example: writeb /3442/0/150 binary:12345.
```
